### PR TITLE
Support zstd ELF compression.

### PR DESCRIPTION
**Pull request by khuey**

### CODE REVIEW

**Comment by khuey** [comment]
@DilumAluthge Can we get libzstd1/libzstd-dev installed on the AArch64 boxes?

**Comment by rocallahan** [comment]
> @DilumAluthge Can we get libzstd1/libzstd-dev installed on the AArch64 boxes?

No need, I have just implemented AWS-based Github Actions CI so we can turn down Buildkite.

**Comment by rocallahan** [comment]
I have just implemented AWS-based Github Actions CI so we can turn down Buildkite and fix this ourselves.

File: CMakeLists.txt [hunk 1]
```diff
@@ -716,6 +725,10 @@ target_link_libraries(rr
   brotli
 )
 
+if(NOT ANDROID)
```
Comment (rocallahan) [inline CMakeLists.txt:728]: slightly cleaner to set a ZSTD flag up above and use that here, instead of duplicating the ANDROID check

**Review by rocallahan — APPROVED** [review]
Looks good with that trivial change

**Pull request merged**